cmd/flags_misc.go

```diff
@@ -40,5 +50,20 @@ func ParseMiscFlags(cmd *cobra.Command) error {
 		logs.SetAllLoggers(level)
 	}
 
+	if ok, err := cmd.Flags().GetBool(pprofFlag); ok && err == nil {
+		// TODO(@Wondertan): Eventually, this should be registered on http server in RPC
+		go func() {
+			mux := http.NewServeMux()
+			mux.HandleFunc("/debug/pprof/", pprof.Index)
+			mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+			mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+			log.Println(http.ListenAndServe("0.0.0.0:6000", mux))
```

Comment:
Port should be user-controllable IMO.

Reply:
True, I would leave this for now and return back once we work through everything that is provided by HTTP. There will be a single port for everything including pprof.

Reply:
https://github.com/celestiaorg/celestia-node/issues/469